tests/schemas/api_data.json

```diff
@@ -625,7 +625,7 @@
     "SharingReencryptedMessageContent": {
         "fields": {
             "author": {
-                "allow_none": true,
+                "allow_none": false,
```

Comment:
should also add author overwritting to `SharingReencryptedMessageContent` `SharingRevokedMessageContent` `PingMessageContent`

Reply:
:+1: